feat(curriculum): add password generator project to Python curriculum (#52444)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/20-upcoming-python/learn-regular-expressions-by-building-a-password-generator/6565c014db6e9b63c257771d.md

---
id: 6565c014db6e9b63c257771d
title: Step 58
challengeType: 20
dashedName: step-58
---

# --description--

> Right before your `for` loop, declare a `count` variable and give it the value of zero.

Reviewer: ```suggestion
Right before your `for` loop, declare a `count` variable and assign the value zero to this variable.
```

# --hints--

> You should declare a `count` variable and give it a value of `0`.

Reviewer: ```suggestion
You should declare a `count` variable and assign the value `0` to this variable.
```
